### ISSUE TITLE
OpenCTI affected with log4 j vulnerability ?

### ISSUE THREAD

**Issue by BITSDevops111** [LuatixHQ/opencti#1775]
https://cybernews.com/news/log4j-vulnerability-a-bombshell-zero-day-exploit-with-global-impact/

**Comment by adel-akloul-mox** [comment]
Anyone can easily mitigate by adding the below in your elasticsearch container option
```
ES_JAVA_OPTS="-Dlog4j2.formatMsgNoLookups=true"
```
Alternatively, bumping ES to `7.16.1` should fix the problem (NOT TESTED): starting Elasticsearch `≥7.16.1`, the Java option that mitigates log4shell is set by default, and the JNDI class is removed, which permanently fix log4shell vulnerability

**Comment by richard-julien** [comment]
OpenCTI doesnt run any java processing so not directly vulnerable. 
Be careful to upgrade any dependencies running with java, like elasticsearch

**Comment by faustus25** [comment]
ElasticSearch just released a new mitigation fix in v7.16.2 available [here](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/deb.html#deb-repo) 

Details on the update log are [here](https://discuss.elastic.co/t/apache-log4j2-remote-code-execution-rce-vulnerability-cve-2021-44228-esa-2021-31/291476)

Upgraded versions should run as:
```
/usr/share/elasticsearch$ sudo ./bin/elasticsearch --version
Version: 7.16.2, Build: default/deb/2b937c44140b6559905130a8650c64dbd0879cfb/2021-12-18T19:42:46.604893745Z, JVM: 17.0.1
```
